apps/meteor/app/e2e/client/rocketchat.e2e.ts

```diff
@@ -437,6 +437,30 @@ class E2E extends Emitter {
 		return decryptedMessageWithAttachments;
 	}
 
+	async decryptPinnedMessage(message: IMessage) {
+		if (!message?.attachments || !message.attachments[0]?.text) {
+			return message;
+		}
+
+		const e2eRoom = await this.getInstanceByRoomId(message.rid);
+		const pinnedMessage = message.attachments[0]?.text;
```

Comment:
Move this to the begining of the func and use the var there to not to chrck twice